next.config.js

```diff
@@ -26,5 +26,14 @@ module.exports = {
     config.resolve.alias['components'] = path.join(__dirname, 'components');
     config.resolve.alias['stores'] = path.join(__dirname, 'stores');
     return config;
-  }
+  },
+
+  async rewrites() {
+    return [
+      {
+        source: '/delegates/:address',
+        destination: '/address/:address',
+      },
+    ];
+  },
```

Comment:
Is this in case someone tries to access via an old URL?

Reply:
yeah, i leave it for now in case some delegate has their profile linked somewhere with the delegates address.